doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -692,35 +678,15 @@ use one or several databases specific to your development.
 Hint locality
 -------------
 
-Hints provided by the ``Hint`` commands are erased when closing a section.
-Conversely, all hints of a module ``A`` that are not defined inside a
-section (and not defined with option ``Local``) become available when the
-module ``A`` is required (using e.g. ``Require A.``).
-
-As of today, hints only have a binary behavior regarding locality, as
-described above: either they disappear at the end of a section scope,
-or they remain global forever. This causes a scalability issue,
-because hints coming from an unrelated part of the code may badly
-influence another development. It can be mitigated to some extent
-thanks to the :cmd:`Remove Hints` command,
-but this is a mere workaround and has some limitations (for instance, external
-hints cannot be removed).
-
-A proper way to fix this issue is to bind the hints to their module scope, as
-for most of the other objects Coq uses. Hints should only be made available when
-the module they are defined in is imported, not just required. It is very
-difficult to change the historical behavior, as it would break a lot of scripts.
-We propose a smooth transitional path by providing the :opt:`Loose Hint Behavior`
-option which accepts three flags allowing for a fine-grained handling of
-non-imported hints.
+To help transitioning hints from the :attr:`global` to the :attr:`export` locality,
+the following option can be useful to approximatively detect hints that are used,
+but would become unavailable after the change of locality:
```

Comment:
```suggestion
The following option can be useful for identifying hints that are used but will become
unavailable when the default hint locality changes from :attr:`global` to :attr:`export` locality
in a future release.  Note that it identifies most but not all such hints.
```

Reply:
As usual, I'm guessing what is intended.  Perhaps you would tweak it's correct?  I thought the wording/sentence structure was convoluted with three clauses.  "Approximately detect" seemed a bit off, too.

Reply:
FWIW, the approximation is the other way around compared to what you wrote, i.e., it identifies hints that are not used. This is explained in the hint description.